test/application_system_test_case.rb

```diff
@@ -4,16 +4,52 @@
 require "capybara/rails"
 require "capybara/minitest"
 
+require "axe/matchers/be_axe_clean"
+require "axe/expectation"
+
 require "test_helpers/cuprite_setup"
 
 Ferrum::Browser.new(process_timeout: 60, timeout: 60)
 
 class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   driven_by :cuprite, using: :chrome, screen_size: [1400, 1400]
 
+  AXE_RULES_TO_SKIP = [:region].freeze
+  AXE_WITHIN_SELECTOR = "body"
+
   def with_preview(preview_name)
     component_uri = self.class.name.gsub("Test", "").gsub("Integration", "").underscore
 
     visit("/rails/view_components/primer/#{component_uri}/#{preview_name}")
+
+    assert_accessible(page)
```

Comment:
Does this mean that all of our existing previews pass Axe?

Reply:
Of the 5 component previews, `Autocomplete` failed. The component renders an input without any labels. I made some updates here to address it but I'm planning to review the component more thoroughly in a follow up PR!